src/System.Diagnostics.TraceSource/src/System.Diagnostics.TraceSource.csproj

```diff
@@ -59,18 +59,21 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="System.Collections" />
-    <Reference Include="System.Collections.NonGeneric" />
-    <Reference Include="System.Collections.Specialized" />
-    <Reference Include="System.ComponentModel.Primitives" />
-    <Reference Include="System.Diagnostics.Debug" />
-    <Reference Include="System.Diagnostics.Process" />
-    <Reference Include="System.Diagnostics.Tools" />
-    <Reference Include="System.IO.FileSystem" />
-    <Reference Include="System.Resources.ResourceManager" />
-    <Reference Include="System.Runtime" />
-    <Reference Include="System.Runtime.Extensions" />
-    <Reference Include="System.Runtime.InteropServices" />
-    <Reference Include="System.Threading" />
+    <ProjectReference Include="..\..\System.Collections\src\System.Collections.csproj" />
```

Comment:
I assume the change from `Reference` to `ProjectReference` is necessary because we are also referencing `System.Private.CoreLib`, and you get duplicate type errors if you reference the `System.Runtime` ref assembly and `System.Private.Corelib`.

/cc @weshaggard